src/modules/main/templates/hooks/useFilters.ts

```diff
@@ -30,6 +29,9 @@ const parseDictionary = (key: 'patternsIds' | 'technologiesIds') => (value: stri
     ? filters
     : { ...filters, [key]: value };
 
+const pipe = (...fns: any[]) => (filters: TemplatesSearchFilters) =>
+  fns.reduce((v, f) => f(v), filters);
```

Comment:
Change fns from any[] to Function[]

Reply:
Changed. I used the search tool and there are no more ramda imports in the project.